docs/book/src/developer/tilt.md

```diff
@@ -100,7 +101,7 @@ extra_args:
 
 ### 1.  Configuration to deploy workload cluster with external cloud controller manager
```

Comment:
This section seems repetitive, I think it can be removed as we made the default template addition in L87 to use CCM.

Reply:
Fixed.

Reply:
I think you removed it in the other file and not here